sdk/messaging/azwebpubsub/internal/version.go

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+//go:build go1.18
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package internal
+
+// Constants to identify the module
+const (
+	// ModuleName is the module name that shows in telemetry.
+	ModuleName = "azwebpubsub"
```

Comment:
We've since updated this to be the full module identity, e.g. `github.com/Azure/azure-sdk-for-go/sdk/messaging/azwebpubsub`